src/JitInterface/src/CorInfoImpl.Intrinsics.cs

```diff
@@ -190,6 +192,12 @@ private CorInfoIntrinsics getIntrinsicID(CORINFO_METHOD_STRUCT_* ftn)
                     }
                     break;
 
+                // TODO: Implementation of pMustExpand in RyuJIT is not correct
```

Comment:
That's right. IsIntrinsicImplementedByUserCall check is relevant for math intrinsic functions only. The check seems wrong. I will follow this up.